packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/value.mapPair.iteration.scope

```diff
@@ -0,0 +1,13 @@
+{ value: 123 }
+---
+
+[#1 Range] =
+[#1 Domain] = 0:0-0:14
+0| { value: 123 }
+  >--------------<
```

Comment:
Huh. Technically this iteration scope is not for this facet. Not sure there's much we can do about that tho 🤷‍♂️

Reply:
I don't think so. This would be a reason to have more fine grain scopes as well.

Reply:
Yeah fair point. I wonder how many of our facets should really be their own scopes 🤔

Reply:
Probably quite a few.

Reply:
I think one clear indicator that a scope should split is that we have multiple iteration scopes as in the above example. For example all the different types of ways you can do functions in javascript is still the same function scope, but a value in a map pair is not really the same thing as a return statement value and they do have different iteration scopes. This is definitely going to be on a case by case basis, but I think that having multiple iteration scopes for the same scope is a bit of a warning sign.